docs/advanced.md

```diff
@@ -92,6 +92,9 @@ app = FastAPI()
 app.add_middleware(WoodchipperFastAPI)
 ```
 
+In either case, you will also need to add `"woodchipper": "INFO:` as an additional logging facility when callling
```

Comment:
I would frame this differently (and also include it for Flask):

> The middleware's logging context is emitted on the `woodchipper.http.fastapi` named facility with `INFO` priority, so you'll want to make sure your logging configuration includes such messages.

Reply:
Sounds good to me. Updated!